eng/pipelines/templates/jobs/ci.yml

```diff
@@ -229,11 +234,23 @@ jobs:
   - template: /eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml
     parameters:
       JobTemplatePath: /eng/pipelines/templates/jobs/ci.tests.yml
+      ProjectListOverrideFile: ${{ parameters.ProjectListOverrideFile }}
       MatrixConfigs: ${{ parameters.MatrixConfigs }}
       MatrixFilters: ${{ parameters.MatrixFilters }}
       MatrixReplace: ${{ parameters.MatrixReplace }}
       CloudConfig:
-        Cloud: Public
+        Cloud: public
+      ${{ if ne(parameters.TestTargetDependency , 'not-specific') }}:
+        SparseCheckoutPaths: 
+          - "/sdk"
+          - "/tools/bootstrapTools/bootstrap.targets"
```

Comment:
What is this bootstrap.targets?

Reply:
It said the `/tools/bootstrapTools/bootstrap.targets` file not found if not introduced. We do not fail the step but has many log messages like this. Will show you what happened.

Reply:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=1732856&view=logs&j=76992fd1-2312-5fae-7c45-7baba86b87ae&t=56d44aed-d89d-5d70-8459-3b40b7070a44

![image](https://user-images.githubusercontent.com/48036328/180875619-524aee19-c458-4298-a765-306d59e13731.png)

I was wrong. It fails the pipeline.

Reply:
This is only used by the older track 1 mgmt libraries. We should make sure those are being excluded and then this shouldn't be an issue.

Reply:
Maybe just default this to include everything except for the recordings.